website/content/en/docs/amazon-sagemaker-integration/sagemaker-components-for-kubeflow-pipelines.md

```diff
@@ -0,0 +1,27 @@
++++
+title = "SageMaker Components for Kubeflow Pipelines"
+description = "Use SageMaker Components for Kubeflow Pipelines with Kubeflow on AWS"
+weight = 20
++++
+
+The [SageMaker Components for Kubeflow Pipelines](https://docs.aws.amazon.com/sagemaker/latest/dg/kubernetes-sagemaker-components-for-kubeflow-pipelines.html) allow you to move your data processing and training jobs from the Kubernetes cluster to SageMaker’s machine learning-optimized managed service. 
+
+These components integrate SageMaker with the portability and orchestration of Kubeflow Pipelines. Using the SageMaker components, each job in the pipeline workflow runs on SageMaker instead of the local Kubernetes cluster. The job parameters, status, logs, and outputs from SageMaker are accessible from the Kubeflow Pipelines UI. 
+
+## Available components
+
+You can create a Kubeflow Pipeline built entirely using SageMaker components, or integrate individual components into your workflow as needed. The following SageMaker components are available:
```

Comment:
instead of listing here, lets just refer to the Github so there is only one single place to manage